snakedeploy/version.py

```diff
@@ -15,7 +15,10 @@
 # Global requirements
 
 
-INSTALL_REQUIRES = (("requests", {"min_version": None}),)
+INSTALL_REQUIRES = (
+    ("requests", {"min_version": None}),
+    ("pandas", {"min_version": None}),
```

Comment:
pandas is a fairly big dependency to add - and known for fairly large changes between it's versions. If you intend to have many other uses for it around the library then we could keep, but if it's just needed to read a csv and shuffle some values around, could the same not be done with the `csv` library (and avoid the heavy dependency?)